config/samples/network_v1beta1_netconfig.yaml

```diff
@@ -21,7 +21,7 @@ spec:
     - name: subnet1
       allocationRanges:
       - end: 172.17.0.250
-        start: 172.17.0.10
+        start: 172.17.0.100
       excludeAddresses:
       - 172.17.0.10
```

Comment:
We need to remove or update these too.

Reply:
oh, that's right, let me remove it. Thx @rabi !